enterprise/frontend/src/embedding-sdk/components/private/PublicComponentWrapper/SdkError.tsx

```diff
@@ -0,0 +1,21 @@
+import { t } from "ttag";
+
+import { useSdkSelector } from "embedding-sdk/store";
+import { getErrorComponent } from "embedding-sdk/store/selectors";
+
+export type SdkErrorProps = { message: string };
+
+export const SdkError = ({ message }: SdkErrorProps) => {
+  const CustomError = useSdkSelector(getErrorComponent);
```

Comment:
Note that putting React components and functions in a Redux store will [break time travel debugging](https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state).

> It is highly recommended that you only put plain serializable objects, arrays, and primitives into your store. It's technically possible to insert non-serializable items into the store, but doing so can break the ability to persist and rehydrate the contents of a store, as well as interfere with time-travel debugging.

> If you are okay with things like persistence and time-travel debugging potentially not working as intended, then you are totally welcome to put non-serializable items into your Redux store. Ultimately, it's your application, and how you implement it is up to you. As with many other things about Redux, just be sure you understand what tradeoffs are involved.

An alternative to this is creating a React context and accessing the components from a React context, rather than putting it in Redux.